### DISCUSSION TITLE
Local development error

### DISCUSSION THREAD

**Discussion by devDoubleH** [prisma/prisma#22449] [Q&A] [answered]
### Question

Hello Everyone! After deploying to vercel my project something went wrong I think because I am getting this error:
```
 Error occurred during query execution:
ConnectorError(ConnectorError { user_facing_error: None, kind: AuthenticationFailed { user: "SCRAM failure: bad auth : authentication failed" }, transient: false })
    at async Page (./app/dashboard/[...folder]/page.tsx:49:16)
 ⨯ PrismaClientUnknownRequestError:
Invalid `prisma.cafe.findUnique()` invocation:
```

### Information about Prisma Schema, Client Queries and Environment (optional)

datasource db {
  provider = "mongodb"
  url      = env("DATABASE_URL")
  relationMode = "prisma"
}

generator client {
  provider = "prisma-client-js"
}

- OS: MacOS Ventura
- Database: MongoDB
- Node.js version: v20.9.0

- Prisma version: 5.7.0

**Comment by ludralph** [comment]
Hi @devDoubleH 👋 

Thank you for raising this question.

The error message suggests that you are having authentication issues with your MongoDB database. Can you please check that your `username` and `password` are correctly applied to your environment variables?

**Comment by devDoubleH** [comment]
Hey @ludralph I checked mongo url twice everything is right

**Comment by devDoubleH** [comment, marked as answer]
So closing this issue 😅. Error was multiple `.env`(s) kind a like `.env.local`     `.env.development.local`. I moved my `DATABASE_URL` all of them then cleaned prisma client generation and regenerated it so then worked thanks @ludralph 🎉.

**Reply by thallysondias** [reply]
same here!

**Comment by AmitKuMaurya** [comment]
I was getting the error.

```
  error: PrismaClientInitializationError:
  Invalid `prisma.template.create()` invocation:
  ```
  
  we were getting that error because the DB was created in schema.prisma should get pushed to prisma client. then it would be initialised for querying.
  push Database_URL which is stored in .env by  command `prisma db push`.
  and again run npx prisma generate.
  
  It worked for me.
